### PR TITLE
callgraph show calls

### PR DESCRIPTION
**Pull request by Domonion**

### CODE REVIEW

File: resharper/resharper-unity/src/CSharp/Daemon/Stages/CallGraphStage/CallGraphAbstractStage.cs [hunk 1]
```diff
@@ -44,52 +49,63 @@ public abstract class CallGraphAbstractStage : CSharpDaemonStageBase
...
             myContextProviders = contextProviders;
-
-            myProblemAnalyzersByContext = problemAnalyzers.GroupBy(t => t.Context)
-                .ToDictionary(t => t.Key, t => t.ToList());
+            myProblemAnalyzers = problemAnalyzers;
         }
 
         public override void Execute(Action<DaemonStageResult> committer)
         {
+            if (myContext.Kind != DaemonProcessKind.GLOBAL_WARNINGS && myContext.Kind != DaemonProcessKind.VISIBLE_DOCUMENT)
```
Comment (krasnotsvetov) [inline resharper/resharper-unity/src/CSharp/Daemon/Stages/CallGraphStage/CallGraphAbstractStage.cs:75]: wrong assumption, that could break find simillar issues action in alt+enter

File: resharper/resharper-unity/src/CSharp/Daemon/Stages/CallGraphStage/CallGraphAbstractStage.cs [hunk 2]
```diff
@@ -44,52 +49,63 @@ public abstract class CallGraphAbstractStage : CSharpDaemonStageBase
...
                 DaemonProcess.ContextBoundSettingsStore);
-            
+
             File.ProcessThisAndDescendants(this, highlightingConsumer);
-            
+
             committer(new DaemonStageResult(highlightingConsumer.Highlightings));
         }
 
+        private void CheckForInterrupt(IHighlightingConsumer consumer)
```
Comment (krasnotsvetov) [inline resharper/resharper-unity/src/CSharp/Daemon/Stages/CallGraphStage/CallGraphAbstractStage.cs:88]: remove and use IsProcessingFinished directly

File: resharper/resharper-unity/src/CSharp/Daemon/Stages/PerformanceCriticalCodeAnalysis/VisualStudio/PerformanceCriticalCodeFormatDefinitions.cs [hunk 3]
```diff
@@ -13,8 +12,6 @@
 namespace JetBrains.ReSharper.Plugins.Unity.CSharp.Daemon.Stages.PerformanceCriticalCodeAnalysis.VisualStudio
 {
     [ClassificationType(ClassificationTypeNames = Name)]
```
Comment (krasnotsvetov) [inline resharper/resharper-unity/src/CSharp/Daemon/Stages/PerformanceCriticalCodeAnalysis/VisualStudio/PerformanceCriticalCodeFormatDefinitions.cs:14]: strange diff

File: resharper/resharper-unity/src/CSharp/Daemon/Stages/UnityHighlightingStage/UnityHighlightingAbstractStage.cs [hunk 4]
```diff
@@ -75,6 +76,9 @@ public class UnityHighlightingProcess : CSharpDaemonStageProcessBase
 
         public override void Execute(Action<DaemonStageResult> committer)
         {
+            if (myContext.Kind != DaemonProcessKind.GLOBAL_WARNINGS && myContext.Kind != DaemonProcessKind.VISIBLE_DOCUMENT)
```
Comment (krasnotsvetov) [inline resharper/resharper-unity/src/CSharp/Daemon/Stages/UnityHighlightingStage/UnityHighlightingAbstractStage.cs:79]: Same problem

File: resharper/resharper-unity/test/data/CSharp/Intentions/QuickFixes/PerformanceAnalysisDisable/Availability/Everything.cs.gold [hunk 5]
```diff
@@ -7,20 +7,20 @@ namespace ContextActionsTests
         private void Update()
         {
             ||GetComponent|(0)|(1)<int>();
-            |PlainExpensiveMethod|(2)();
-            |CommentExpensiveMethod|(3)();
-            |TransitiveExpensiveeMethod|(4)();
+            ||PlainExpensiveMethod|(2)|(3)();
```
Comment (krasnotsvetov) [inline resharper/resharper-unity/test/data/CSharp/Intentions/QuickFixes/PerformanceAnalysisDisable/Availability/Everything.cs.gold:10]: Double expensive higlighiting, looks like wrong gold

File: resharper/resharper-unity/test/data/CSharp/Intentions/QuickFixes/PerformanceAnalysisDisable/Availability/Everything.cs.gold [hunk 6]
```diff
@@ -7,20 +7,20 @@ namespace ContextActionsTests
...
+            ||CommentExpensiveMethod|(4)|(5)();
```
Comment (krasnotsvetov) [inline resharper/resharper-unity/test/data/CSharp/Intentions/QuickFixes/PerformanceAnalysisDisable/Availability/Everything.cs.gold:11]: same here

**Comment by Domonion** [comment]
https://github.com/JetBrains/resharper-unity/pull/1995

**Pull request closed without merging**